### PR TITLE
Add Restrictions for Reserved Addresses

### PR DESCRIPTION
**Pull request by matbrgz**
The IANA reserve some emails to NICs.

E.g.
iana-admin@dnsbelgium.be (as see [here](https://www.iana.org/domains/root/db/be.html))
tld-admin@nic.pe

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -1098,6 +1171,13 @@
   "tj",
   "tk",
   "tl",
+  "tld",
+  "tld-admin",
+  "tld-tech",
+  "tld.admin",
+  "tld.ops",
```
Comment (matbrgz) [inline index.json:1179]: Missing tld.tech

File: admin-list.json [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+  "adm"
   "admin",
   "administrator",
-  "webmaster",
   "hostmaster",
   "postmaster",
-  "ssl"
+  "security"
+  "ssl",
+  "webmaster"
```
Comment (matbrgz) [inline admin-list.json:11]: Missing comma (,)

**Comment by titanism** [comment]
Please rebase this into one commit, thank you.

**Comment by matbrgz** [comment]
I try to do it on my end, don't know exactly if it works.

But you can rebase and marge when accepting the pull request according to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges

**Comment by titanism** [comment]
Please rebase on your end, as the rebase commit preview is not a feature (we cannot preview the entire commit without having to confirm the merge on our side resulting in a commit to this repository without preview).

**Comment by titanism** [comment]
The IANA ones added should have also been added to `admin-list.json`, but we will correct this.  Thanks!

**Comment by titanism** [comment]
Released in v1.0.0 to npm and GitHub

https://github.com/forwardemail/reserved-email-addresses-list/releases/tag/v1.0.0

**Pull request merged**